### ISSUE TITLE
Request: Schweinfurt, Germany

### ISSUE THREAD

**Issue by UeEiFetischist** [mampfes/hacs_waste_collection_schedule#664]
Could you please add support of Schweinfurt (city)?
Link to the source: [https://www.schweinfurt.de/leben-freizeit/umwelt/abfallwirtschaft/4427.Aktuelle-Abfuhrtermine-und-Muellkalender.html](https://www.schweinfurt.de/leben-freizeit/umwelt/abfallwirtschaft/4427.Aktuelle-Abfuhrtermine-und-Muellkalender.html)

Thanks in advance!

**Comment by bbr111** [comment]
Shoud not be a problem.
the data is in json format.
an other option could be the ICAL link at the bottom of the page.

**Comment by bbr111** [comment]
i'm able to catch the json data with an test script.
But is it right, das all waste types are collcted at the same date?

```
2023-04-26 00:00:00 2023-04-26 00:00:00 Papier, Restmüll, Gelber Sack
2023-05-06 08:00:00 2023-05-06 08:00:00 Mobiler Wertstoffhof und Problemabfallsammlung - Parkplatz Galgenleite
2023-05-10 00:00:00 2023-05-10 00:00:00 Papier, Restmüll, Gelber Sack
2023-05-24 00:00:00 2023-05-24 00:00:00 Papier, Restmüll, Gelber Sack
2023-06-03 08:00:00 2023-06-03 08:00:00 Mobiler Wertstoffhof und Problemabfallsammlung - Goethestraße
2023-06-07 00:00:00 2023-06-07 00:00:00 Papier, Restmüll, Gelber Sack
2023-06-21 00:00:00 2023-06-21 00:00:00 Papier, Restmüll, Gelber Sack
2023-07-01 08:00:00 2023-07-01 08:00:00 Mobiler Wertstoffhof und Problemabfallsammlung - Lindenbrunnenweg

```

And what to do with "Mobiler Wertstoffhof und Problemabfallsammlung" ?

**Comment by UeEiFetischist** [comment]
Thank you! 
Yes, all types of waste are collected at the same data.

In my opinion you can remove "Mobiler Wertstoffhof und Problemabfallsammlung" because it is not necessary for your home waste.

**Comment by bbr111** [comment]
solved with pr #669 

HappyHippo :-)

**Comment by mampfes** [comment]
fixed by #669
